pkg/breakglasscredential/flags.go

```diff
@@ -0,0 +1,144 @@
+package breakglasscredential
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"time"
+
+	cmv1 "github.com/openshift-online/ocm-sdk-go/clustersmgmt/v1"
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+
+	"github.com/openshift/rosa/pkg/interactive"
+	"github.com/openshift/rosa/pkg/rosa"
+)
+
+const (
+	usernameFlag   = "username"
+	expirationFlag = "expiration"
+)
+
+type BreakGlassCredentialArgs struct {
+	username           string
+	expirationDuration time.Duration
+}
+
+func AddBreakGlassCredentialFlags(cmd *cobra.Command, prefix string) *BreakGlassCredentialArgs {
```

Comment:
@chenz4027 The `prefix` parameter to this function is not being used, so I think lets remove it 👍